gsoc2018-bharat/src/sparql.py

```diff
@@ -0,0 +1,73 @@
+from SPARQLWrapper import SPARQLWrapper, JSON
+import asyncio
```

Comment:
also have the feeling that i saw this somewhere else. Is this currently used to get training / eval data ?

Reply:
It was used to get the eval data for t-SNE. I used the data to plot entities in the graphs that I shared on the slack channel.